main.py

```diff
@@ -1,7 +1,68 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+
+    Actor.objects.create(
+        first_name="George",
+        last_name="Klooney",
+    )
+    Actor.objects.create(
+        first_name="Kianu",
+        last_name="Reaves",
+    )
+    Actor.objects.create(
+        first_name="Scarlett",
+        last_name="Keegan",
+    )
+    Actor.objects.create(
+        first_name="Will",
+        last_name="Smith",
+    )
+    Actor.objects.create(
+        first_name="Jaden",
+        last_name="Smith",
+    )
+    Actor.objects.create(
+        first_name="Scarlett",
+        last_name="Johansson",
+    )
+
+    Genre.objects.filter(
+        name="Dramma"
+    ).update(
+        name="Drama"
+    )
+    Actor.objects.filter(
+        last_name="Klooney"
+    ).update(
+        last_name="Clooney"
+    )
+    Actor.objects.filter(
+        first_name="Kianu"
+    ).update(
+        first_name="Keanu",
+        last_name="Reeves"
+    )
+    Genre.objects.filter(
+        name="Action"
+    ).delete()
+    Actor.objects.filter(
```

Comment:
Would make sense to break down main function is smaller specialized functions to structure your code better. E.g separate function for creating, updating, deleting..